pkg/oc/cli/admin/policy/modify_scc.go

```diff
@@ -14,8 +14,7 @@ import (
 	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	"k8s.io/kubernetes/pkg/kubectl/genericclioptions"
 
-	securityclient "github.com/openshift/client-go/security/clientset/versioned"
-	securitytypedclient "github.com/openshift/client-go/security/clientset/versioned/typed/security/v1"
+	securityv1client "github.com/openshift/client-go/security/clientset/versioned/typed/security/v1"
```

Comment:
@deads2k proposed the following convetions:
```
<group><version> api/group/version
<group><version>conversions internalrepo/pkg/apis/group/version
<group><version>client|lister|informer
<group><version>typedclient deeply nested client type
```
which means this should be `securityv1typedclient`

Reply:
If you want to you can update our coding conventions with these ;)

Reply:
Mind if I do this update as a followup? I think it might make more sense to do a sweep of all of the commands in a future effort to switch to these new conventions